ding/model/template/q_learning.py

```diff
@@ -806,16 +810,25 @@ def forward(self, inputs: Dict, inference: bool = False, saved_state_timesteps:
             x['next_state'] = next_state
             return x
         else:
+            # NOTE(rjy): in r2d2, 
```

Comment:
why add R2D2 information here

Reply:
without r2d2, I can't understand why we need saved_state here and which state should be saved

Reply:
add this reason in comments here